### PR TITLE
Add docker swarm service

### PR DESCRIPTION
**Pull request by tdobrovolny**
Docker swarm service allow communication between docker engines in
swarm mode.

### CODE REVIEW

**Comment by erig0** [comment]
You also need to update config/Makefile.am

File: config/services/docker-swarm.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Docker swarm</short>
+  <description>Ports and protocol required by docker in swarm mode</description>
```
Comment (erig0) [inline config/services/docker-swarm.xml]: This should be a one line description of Docker Swarm not a description of the service file.

Comment (tdobrovolny) [reply config/services/docker-swarm.xml]: TCP port 2376 is not required by docker engines running in swarm mode. See https://docs.docker.com/engine/swarm/swarm-tutorial/#open-protocols-and-ports-between-the-hosts

Your link describes legacy standalone swarm used on docker prior the version 1.12. See https://docs.docker.com/swarm/provision-with-machine/

So if you want also open port 2376 needed for docker-machine and legacy swarm, I recommend have two firewalld services. One for "Docker legacy Swarm" also with port 2376 and one for "Docker integrated Swarm" without 2376 port.

**Comment by tdobrovolny** [comment]
TCP port 2376 is not required by docker engines running in swarm mode. See https://docs.docker.com/engine/swarm/swarm-tutorial/#open-protocols-and-ports-between-the-hosts

Your link describes legacy standalone swarm used on docker prior the version 1.12. See https://docs.docker.com/swarm/provision-with-machine/

So if you want also open port 2376 needed for docker-machine and legacy swarm, I recommend have two firewalld services. One for "Docker legacy Swarm" also with port 2376 and one for "Docker integrated Swarm" without 2376 port.

**Comment by FruityWelsh** [comment]
This TCP port is also used by Rancher nodes as per the [Rancher req docs](https://rancher.com/docs/rancher/v2.x/en/installation/requirements/). For "Docker daemon TLS port used by Docker Machine (only needed when using Node Driver/Templates)".

**Pull request merged**